models/migrations/v204.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2021 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"fmt"
+
+	"xorm.io/xorm"
+)
+
+func addAuthorizeColForTeamUnit(x *xorm.Engine) error {
+	type TeamUnit struct {
+		ID        int64 `xorm:"pk autoincr"`
+		OrgID     int64 `xorm:"INDEX"`
+		TeamID    int64 `xorm:"UNIQUE(s)"`
+		Type      int   `xorm:"UNIQUE(s)"`
+		Authorize int
```

Comment:
```suggestion
		AccessMode int
```

Reply:
This cannot be changed because the column on table hasn't been changed.